NAS-102000: Update text about configuring multiple IP addresses (#1451)
freenas/freenas-docs: userguide/network.rst

   |                     |                |             |                                                                                                           |
   +---------------------+----------------+-------------+-----------------------------------------------------------------------------------------------------------+
   | IP Address          | integer and    | All         | Static IPv4 or IPv6 address and subnet mask. Example: *10.0.0.3* and *22*. Click                          |

>    |                     | drop-down menu |             | :guilabel:`ADD` to set multiple IP addresses. Clicking :guilabel:`DELETE` removes the related             |

Reviewer: "set" is an unclear verb here.
```suggestion
   |                     | drop-down menu |             | :guilabel:`ADD` to add another IP address. Click :guilabel:`DELETE` on an IP address to delete it.             |
```
